### DISCUSSION TITLE
different browser display

### DISCUSSION THREAD

**Discussion by DennisXK** [arendst/Tasmota#21904] [Q&A]
Why are there different presentations between browsers? 
![Edge](https://github.com/user-attachments/assets/13fe6be9-f302-43f3-9c00-6edc1451edf0)
Without the clock it is almost impossible to set the time.

**Comment by arendst** [comment]
That's because new browsers introduce new features in a crippled way leading to a too small entry box.

**Comment by DennisXK** [comment]
If I make the input field larger, it's exactly like that

You can also see a difference on the second button.

**Comment by sfromis** [comment]
As what you show is not how Tasmota works "out of the box", things also depends on what you did to get those extra fields.

**Comment by DennisXK** [comment]
I created the input fields with Tasmota script. I assume the tasmota script is “out of the box”.

**Reply by sfromis** [reply]
Not really. The scripter feature is standard for Tasmota, but when you code something that way, this is not "out of the box", but up to what you code.
